python/py-halo/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-halo
+version             0.0.31
+revision            0
+
+license             MIT
+supported_archs     noarch
+platforms           {darwin any}
+
+maintainers         {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description         Beautiful terminal spinners in Python
+long_description    {*}${description}
+
+homepage            https://github.com/manrajgrover/halo
+
+distname            halo-${version}
+
+checksums           rmd160  2a67ba23ee8883c2ba3c4870a64ce2201d2d4819 \
+                    sha256  7b67a3521ee91d53b7152d4ee3452811e1d2a6321975137762eb3d70063cc9d6 \
+                    size    11666
+
+python.versions     37 38 39 310
```

Comment:
perhaps only add recent version at this time unless there are other packages that depend on older subports. I'd recommend "310 311" only.

Reply:
fixed